File: src/main/java/com/manning/simplysend/mappers/CredentialsMapper.java
package com.manning.simplysend.mappers;

import com.manning.simplysend.dto.UserDTO;
import com.manning.simplysend.entities.UserCredentials;

public class CredentialsMapper {

    public static UserCredentials from(UserDTO userDTO) {
        UserCredentials uc = new UserCredentials();
        uc.setPassword(userDTO.getPassword());
        return uc;
    }
}

File: src/main/kotlin/com/manning/simplysend/mappers/CredentialsMapper.kt
package com.manning.simplysend.mappers

import com.manning.simplysend.dto.UserDTO
import com.manning.simplysend.entities.UserCredentials

object CredentialsMapper {
    
    fun from(userDTO: UserDTO): UserCredentials {
        val uc = UserCredentials()
        uc.password = userDTO.password
        return uc
    }
}